doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -199,20 +206,24 @@ If needed, the format can be explicitly set with the :option:`--format` option.
    python -m robot.libdoc OperatingSystem OperatingSystem.xml
    python -m robot.libdoc test/resource.robot doc/resource.libspec
    python -m robot.libdoc --format xml MyLibrary MyLibrary.spec
-   python -m robot.libdoc --format xml:html MyLibrary MyLibrary.xml
+   python -m robot.libdoc --format xml -s html MyLibrary MyLibrary.xml
 
 The exact Libdoc spec file format is documented with an `XML schema`__ (XSD)
 at https://github.com/robotframework/robotframework/tree/master/doc/schema.
-The spec file format may change slightly between Robot Framework major
-releases. To make it easier for external tools to know how to parse a certain
+The spec file format may change between Robot Framework major releases.
+To make it easier for external tools to know how to parse a certain
 spec file, the spec file root element has a dedicated `specversion`
 attribute. It was added in Robot Framework 3.2 with value `2` and earlier
 spec files can be considered to have version `1`. The spec version will
 be incremented in the future if and when changes are made.
+Robot Framework 4.0 introduced new spec version `3`.
 
-.. note:: The `xml:html` format and automatically using it if the output
-          file extension is :file:`*.libspec` are new features in Robot
-          Framework 3.2.
+.. note:: Robot Framework 4.0 introduced an updated xml format. It is
+          incompatible with former versions. LibDoc can only read and write
+          new XML format.
+
+          The `xml:html` format introduced in Robot Framework 3.2. has been
+          replaced by the option `-s --specdocformat <raw|html>`.
```

Comment:
We have separate `:option:` role that should be used here. We've also only used the long option name, no short option or values, in cases like this. In other words, this should be `:option:'--specdocformat'` (with single quotes replaced with backticks).